tests/test_other.py

```diff
@@ -225,7 +225,11 @@ def test_emcc_2(self):
           print('(no output)')
           continue
         syms = Building.llvm_nm(target)
-        assert len(syms.defs) == 1 and ('main' in syms.defs or '__original_main' in syms.defs), 'Failed to generate valid bitcode'
+        assert 'main' in syms.defs
+        if self.is_wasm_backend():
+          assert len(syms.defs) == 2 and '__original_main' in syms.defs
+        else:
+          assert len(syms.defs) == 1
```

Comment:
The name `__original_main` seems like a detail we probably don't want to care about here.

How about just "assert len(syms.defs) <= 2"?

Reply:
I agree the name is too detailed, removing. I think it's useful to keep `== 2` in this path though, so we are alerted if this changes.